ui/changes/src/index.js

```diff
@@ -1081,11 +1078,20 @@ function setTableHeaderHandlers() {
             setOption(optionName, radio.value);
           }
         }
-        rebuildTable();
+        renderUI();
       };
     } else {
       throw new Error("Unexpected option type.");
     }
   });
-  buildTable();
+
+  if (localStorage.getItem("detailsToggle") !== null) {
+    bugDetails.open = !!JSON.parse(localStorage.getItem("detailsToggle"));
+  }
+  bugDetails.addEventListener("toggle", () => {
+    localStorage.setItem("detailsToggle", bugDetails.open);
+    renderUI(false);
+  });
+
+  renderUI();
 })();
```

Comment:
Nit: you could use localForage to avoid synchronous IO.

Reply:
Yeah I started on that and then realized we were setting the config to indexeddb in common.js and didn't know if that would race with a different driver here. Guess we could used indexeddb here, although in that case if we went async we'd have to wait to render until we get the result anyway. I don't feel super strongly either way, fine to update it.

Reply:
Also I believe indexeddb doesn't work in private browsing windows so we'd lose the UI state there (again, we could use localStorage backend for localForage it just seemed like this was a global state based on https://github.com/mozilla/bugbug/blob/27512d1a8f25a602abd3aaa100e2bf0f10ef4647/ui/changes/src/common.js#L19 without digging into the docs

Reply:
Yeah I was thinking of using IndexedDB. I always try to avoid localStorage because of its sync IO (even with a small object to store like this one, it can always cause jank).

Reply:
Found a way to namespace the storage in the config (https://localforage.github.io/localForage/#settings-api-config) just to be safe, and updated the code to use localForage / indexeddb